app/views/site/edit.html.erb

```diff
@@ -1,8 +1,12 @@
 <% content_for :content do %>
   <% if Settings.status == "database_offline" or Settings.status == "api_offline" %>
+    <div class="alert alert-danger text-center">
     <p><%= t "layouts.osm_offline" %></p>
+    </div>
   <% elsif Settings.status == "database_readonly" or Settings.status == "api_readonly" %>
+    <div class="alert alert-danger text-center">
```

Comment:
Offline and read_only modes are not "danger" messages, since there's nothing that the user can or has done wrong. 

If we look at the `map_layout` helper in https://github.com/openstreetmap/openstreetmap-website/blob/e443d99edd9c9b68d6c0fa41b028bb9a5cdb07c5/app/controllers/application_controller.rb#L258-L263 we can see that both are treated as `warning` messages